tests/platform_tests/test_cpu_memory_usage.py

```diff
@@ -84,26 +84,31 @@ def test_cpu_memory_usage(duthosts, enum_rand_one_per_hwsku_hostname, setup_thre
                             outstanding_procs_counter, proc)
 
     analyse_monitoring_results(cpu_threshold, memory_threshold, outstanding_mem_polls, outstanding_procs,
-                               outstanding_procs_counter, persist_threshold)
+                               outstanding_procs_counter, persist_threshold, monit_result)
 
 
 def analyse_monitoring_results(cpu_threshold, memory_threshold, outstanding_mem_polls, outstanding_procs,
-                               outstanding_procs_counter, persist_threshold):
+                               outstanding_procs_counter, persist_threshold, monit_result):
```

Comment:
It's more clear to extract the current cpu/mem usage outside of this function.
Make this parameters of this function like 
```

def analyse_monitoring_results(cpu_threshold, memory_threshold, outstanding_mem_polls, outstanding_procs,
                               outstanding_procs_counter, persist_threshold, cpu_used_percent, memory_used_percent):
```